File: Java/Car.java
public class Car {
    Integer id;
    String license;
    Account driver;
    private Integer passenger;

    public Car(Account driver, String license){
        this.license = license;
        this.driver = driver;
    }

    public void setPassenger(Integer passenger){
        if(passenger == 4){
            this.passenger = passenger;
        }else{
            System.out.println("You need 4 available seats");
        }
    }

    public Integer getPassenger(){
        return passenger;
    }

    void printDataCar () {
        System.out.println("Driver: " + driver.name + " Document: " + driver.document +" License: " + license + " passenger: " + passenger);
    }
}

File: JS/Car.js
class Car {
    constructor(driver, license, passenger) {
    this.id;
    this.license = license;
    this.driver = driver;
    this.passenger = passenger;
    }

    printDataCar() {
    console.log(this.driver);
    console.log(this.driver.name);
    console.log(this.driver.document);
    console.log(this.license);
    console.log(this.passenger);
    }
}